packages/web/src/components/common/notification-list/NotificationList.tsx

```diff
@@ -10,11 +10,29 @@ import { TransactionGroupsType } from "@components/common/notification-button/No
 
 interface NotificationListProps {
   txsGroupsInformation: TransactionGroupsType[];
+  modalRef: React.RefObject<HTMLDivElement | null>;
+  modalOutSideClick: () => void;
```

Comment:
It's a small thing, but I think function names should start with a verb.
If it's an event function, it looks good to start with on.

Reply:
I agree. I'll fix it.